main.go

```diff
@@ -342,10 +347,10 @@ func createDeploymentCacheConfig(platform cluster.Platform) map[string]cache.Con
 	case cluster.ManagedRhods: // no need workbench NS, only SFS no Deployment
 		namespaceConfigs["redhat-ods-monitoring"] = cache.Config{}
 		namespaceConfigs["redhat-ods-applications"] = cache.Config{}
-		//TODO: if ModelReg has a RHOAI NS
+		namespaceConfigs["rhoai-model-registries"] = cache.Config{}
 	case cluster.SelfManagedRhods:
 		namespaceConfigs["redhat-ods-applications"] = cache.Config{}
-		//TODO: if ModelReg has a RHOAI NS
+		namespaceConfigs["rhoai-model-registries"] = cache.Config{}
```

Comment:
@zdtsw  I guess we can use the `modelregistry.DefaultModelRegistriesNamespace` variable as it is going to be amended for downstream ?

Reply:
yep, i have some tests ongoing and will have this updated.

Reply:
updated